include/boost/geometry/algorithms/detail/envelope/point.hpp

```diff
@@ -38,9 +38,11 @@ namespace detail { namespace envelope
 struct envelope_point
 {
     template <typename Point, typename Box, typename Strategy>
-    static inline void apply(Point const& point, Box& mbr, Strategy const& )
+    static inline void apply(Point const& point, Box& mbr, Strategy const& strategy)
     {
-        Strategy::apply(point, mbr);
+        // strategy.envelope(point, mbr).apply(point, mbr);
+        using strategy_t = decltype(strategy.envelope(point, mbr));
+        strategy_t::apply(point, mbr);
```

Comment:
It looks good.
But this way strategy state is lost. That might of course be solveable, but it might be good to have an example of that.

Reply:
Yes, I simply replaced the already existing line. It was and it is implemented like that because in case of points the only information that is needed is the CS because envelope for points doesn't use radius nor ellipsoid. But yes, if we want to be fully correct then the line that is currently commented out should be used instead.

Reply:
I tested with `area` and indeed, the strategy is not lost, so it works really well